custom/conf/app.example.ini

```diff
@@ -78,6 +78,9 @@ RUN_MODE = ; prod
 ;; Domain name to be exposed in clone URL
 ;SSH_DOMAIN = %(DOMAIN)s
 ;;
+;; User to be exposed in clone URL. If blank, then it is the value of BUILTIN_SSH_SERVER_USER.
+;SSH_USER =
```

Comment:
Could you match what you have in the markdown docs here `%(...`

Reply:
The main reason I didn't was because the style for the variables in both places is inconsistent, and I was mostly copying what I saw. Can still make the change if you think it'll be better that way

Reply:
cc @techknowlogick since it's been a few days, just wanna ask whether you want me to make the change or not

Reply:
Apologies for the disappearance.

Could you make these changes? I know that there are inconsistencies between these two files, although hopefully they will be aligned, and to make the effort of that future person less that is why I'm requesting they be aligned.

I don't have super strong feelings about this, and so I will add a LGTM and leave it up to you if you wish to change it so I'm not blocking this PR from being merged :)

Reply:
Apologies for my disappearance!

Reading this again, I'm not 100% sure what you meant by copying the markdown docs, so, I did what made the most sense to me and put the default directly as the variable value. Also did the same for `SSH_BUILTIN_SERVER_USER` as well since this directly uses that. Let me know if this is what you'd prefer.